app/javascript/components/widgets/weather/semi-circle.jsx

```diff
@@ -36,6 +42,8 @@ const getOrbPercent = (endTime, totalTime, offset) => {
 
 const blue = '#6bd9fe';
 const lightBlue = '#8096ff';
+const yellow = '#fbd273';
+const lightYellow = '#FAE7A7';
```

Comment:
lets keep all new colors in the theme also

Reply:
I asked that they be split out. These are more related to the specific component than an actual theme which lead to confusing/unhelpful names like `yellow2`.

Reply:
Colors should be consistent across the entire app. It doesn't make sense to have single use colors. Can we change the PVD palette to match the Boulder one?

`lightYellow` seems fine, certainly better than `yellow2`. Another option would be to name the color after its function, and not what it looks like (primary/secondary/etc)

Reply:
Check out the original changes. There's `yellow`, `lightYellow` and `yellow2`. Fallyn discovered most of the colors are in fact specific to this component so there's not much advantage to co-locating them for the sake of co-locating them. We could namespace in the theme with something like:
```
colors: {
  city: {
    yellow: ...
    lightYellow: ...
  }
  yellow: ...
}
```
but now we're back to why should we co-locate colors that only related to the city image in a theme file?

Changing the color palette of PVD feels out of scope of creating a new BLD image.

Reply:
The purpose of colocating theme colors is so that they can be changed easily. Maybe plug them into the theme with the city name, and we can drop all the extra logic. How many colors does this component need?

```js
colors: {
  Boulder: [yellow1, yellow2, yellow3],
  Providence: [...]
  ...
}
```

Would be nice to extract the colors from this SVG also, but it looks like our other images are PNGs anyway so it wouldn't be super helpful.

Reply:
We're back to colocating colors that are used in one place. How about:
```
const Boulder = { /* colors */ };
const Default = { /* colors */ };
```
at the top of the component?

Reply:
And those would be something like:
```
const Boulder = { sun: "...", moon: "...", foreground: "....", background: "...."  };
```

Reply:
> We're back to colocating colors

Yes, exactly this. This is what I'm asking for.

Reply:
This should be fixed now!